### PR TITLE
Rc 0.2.0

### LINKED ISSUE #55: setup links are incorrectly pointing to python-metar
All links in our packaging point away from this project towards a dependent project `python-metar`.  The next release will correct this to point to this project instead per standard packaging expectations and pypi links. Users can go find `python-metar` if they wish, but this project's url should point to this project.

We will continue to credit `python-metar` extensively throughout our readme, docs, and docstrings.

### PR DESCRIPTION
**Pull request by austinorr**
This PR is intended to prep the package for a public release.

Goals are:
* installable with minimal requirements (pytest is not/should not be required). test that package works with minimal pip install during CI
* declare requirements and dev-requirements in setup.cfg
* update setup.py to track versions correctly in preparation for pypi release.
* drop unmaintained and unneeded developer & CI tools (generate_vtasks.py, check_cloudside.py, travis)
* consolidate CI actions that have the same triggers
* improve flake8 style and black format conformance
* update Readme (and code where needed) to include a working example.

closes #55, #54, #53, #52, #46

### CODE REVIEW

File: .github/workflows/python-runtests-basic.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 pytest
-        if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Test with pytest
       run: |
-        python check_cloudside.py --runslow
+        pytest -x
```
Comment (phobson) [inline .github/workflows/python-runtests-basic.yml:33]: The `-x` flag here will stop the test on the first failure. Is that what you'd like?

File: cloudside/tests/test_viz.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 
 IMG_OPTS = dict(
-    tolerance=int(os.environ.get("MPL_IMGCOMP_TOLERANCE", 25)),
+    tolerance=int(os.environ.get("MPL_IMGCOMP_TOLERANCE", 37)),
```
Comment (phobson) [inline cloudside/tests/test_viz.py:17]: I've pretty come around to the idea that image testing is a fool's errand for anyone but put viz libraries.

**Review by phobson — APPROVED** [review]

File: cloudside/tests/test_viz.py (hunk 2 above)
Comment (austinorr) [reply cloudside/tests/test_viz.py:17]: perhaps you're right, but let's drop these in a patch release. For today I'm just tidying up what's already here. I also think the --mpl image check is really cool and helps us find unexpected bugs.

File: .github/workflows/python-runtests-basic.yml (hunk 1 above)
Comment (austinorr) [reply .github/workflows/python-runtests-basic.yml]: no, you're right it's better to get a full list of test failures from CI. I switched this to -v instead.

**Review by phobson — APPROVED** [review]

**Comment by phobson** [comment]
Looks good. Do you want the honors of merging?

**Comment by austinorr** [comment]
closes #55
closes #54
closes #53
closes #52
closes #46

**Pull request merged**